packages/styles/scss/components/footer/_footer-nav.scss

```diff
@@ -54,6 +55,7 @@
 
 @include exports('footer-nav') {
   @include carbon--theme($carbon--theme--g90) {
+    @include accordion;
```

Comment:
@annawen1 I'm thinking through this, I'm wondering if we should scope these styles, so we would have these in the corresponding wrapper class to be sure, and that this doesn't affect other accordions or buttons outside of the component:

```
.${prefix}--footer {
  @include accordion;
  @include button;
}
```

Reply:
ah ok sure